Jenkinsfile

```diff
@@ -3,88 +3,126 @@ def isMainBranch() { env.BRANCH_NAME == 'main' }
 def isReleaseBranch() { env.BRANCH_NAME.startsWith('release/') }
 def isFeatureBranch() { env.BRANCH_NAME.startsWith('feature/') }
 
+def oktaproperties = ''
+
 void setBuildStatus(String message, String state) {
   step([
       $class: "GitHubCommitStatusSetter",
       reposSource: [$class: "ManuallyEnteredRepositorySource", url: "https://github.com/MyPureCloud/genesys-messenger-transport-mobile-sdk"],
       contextSource: [$class: "ManuallyEnteredCommitContextSource", context: "ci/jenkins/build-status"],
       errorHandlers: [[$class: "ChangingBuildStatusErrorHandler", result: "UNSTABLE"]],
       statusResultSource: [ $class: "ConditionalStatusResultSource", results: [[$class: "AnyBuildResult", message: message, state: state]] ]
-  ]);
+  ])
 }
 
-pipeline{
-    agent{
+pipeline {
+    agent {
         label "mobile-sdk-dev"
     }
-    options{
+    options {
         parallelsAlwaysFailFast()
     }
     environment {
         DEPLOYMENT_ID = credentials("messenger-mobile-sdk-deployment-id")
         DEPLOYMENT_DOMAIN = 'inindca.com'
-        OKTA_DOMAIN = 'dev-2518047.okta.com'
-        CLIENT_ID = credentials("messenger-mobile-sdk-okta-client-id")
-        SIGN_IN_REDIRECT_URI = 'com.okta.dev-2518047://oauth2/code'
-        SIGN_OUT_REDIRECT_URI = 'com.okta.dev-2518047:/'
-        OKTA_STATE = credentials("messenger-mobile-sdk-okta-state")
-        CODE_CHALLENGE = 'Cc6VZuBMOjDa9wKlFZLK-9lLPr_Q5e7mJsnVooFnBWA'
-        CODE_CHALLENGE_METHOD = 'S256'
-        CODE_VERIFIER = 'BtNSLgCNFlZPEOodtxgIp7c-SlnC0RaLilxRaYuZ7DI'
-        HOME = """${sh(
-            returnStdout: true,
-            script: 'if [ -z "$HOME" ]; then echo "/Users/$(whoami)"; else echo "$HOME"; fi'
-        ).trim()}"""
     }
-    stages{
-        stage("Prepare"){
-            steps{
+    stages {
+        stage("Prepare") {
+            steps {
                 sh 'printenv | sort'
                 setBuildStatus("Preparing", "PENDING")
             }
         }
-        stage("CI Static Analysis"){
-            parallel{
-                stage("Lint"){
-                    steps{
+
+        stage('Get okta.properties') {
+                    agent {
+                        node {
+                            label 'dev_mesos_v2'
+                            customWorkspace "jenkins-mtsdk-${currentBuild.number}"
+                        }
+                    }
+                    steps {
+                        script {
+                            def pipelineLibrary = library('pipeline-library').com.genesys.jenkins
+                            def testing = pipelineLibrary.Testing.new()
+                            oktaproperties = testing.getSecretStashSecret(
+                                'dev',
+                                'us-east-1',
+                                'mobiledx-ios',
```

Comment:
I saw comment in the chat:
`I think we should get the transportsdk stash here instead of mobiledx-ios.`
and we still have `mobiledx-ios`.
What was the resolution?

Reply:
> I saw comment in the chat: `I think we should get the transportsdk stash here instead of mobiledx-ios.` and we still have `mobiledx-ios`. What was the resolution?

I agree with you. This is a temporary workaround.
We decided to handle it separately in the next sprint: https://inindca.atlassian.net/browse/MTSDK-688
For now we can fetch okta.properites from mobiledx-ios group.